### PR TITLE
Adds `Ambivalent` theme + minor refactor of `aquarel/utils.py`

### PR DESCRIPTION
**Pull request by saforem2**
- Add [`ambivalent`](https://github.com/saforem2/ambivalent) theme:

  ambivalent

- Pull out method for creating plot into new function, `make_graph()` in `aquarel/utils.py`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -113,6 +113,7 @@ Add your own with a pull request!
 
 | Name            | Description                                                                                      | Preview                                                                              |
 | :-------------- | :----------------------------------------------------------------------------------------------- | :----------------------------------------------------------------------------------- |
+| [`ambivalent`](https://github.com/saforem2/ambivalent)   | Transparent backgrounds with grey text $\textcolor{#838383}{\blacksquare}$ that are accessible / legible and `{light, dark}`-mode independent | ![](https://github.com/saforem2/aquarel/blob/main/assets/ambivalent.png?raw=true)   |
```
Comment (saforem2) [inline README.md:116]: Add `ambivalent` to examples in README

File: aquarel/__init__.py [hunk 2]
```diff
@@ -1,2 +1,9 @@
 from .theme import Theme
-from .utils import load_theme, list_themes
+from .utils import load_theme, list_themes, make_graph
```
Comment (saforem2) [inline aquarel/__init__.py:2]: - add `aquarel.utils.make_graph` to `aquarel/__init__.py`
- Set `__all__` for wildcard `*` imports

File: aquarel/themes/ambivalent.json
Comment (saforem2) [inline aquarel/themes/ambivalent.json:1]: Adds `aquarel/themes/ambivalent.json`

File: aquarel/utils.py [hunk 3]
```diff
@@ -1,7 +1,15 @@
...
 from .theme import Theme
-from typing import Union
+from typing import Optional, Union
+from pathlib import Path
+
+import matplotlib.pyplot as plt
+import seaborn as sns
+
+HERE = Path(__file__).parent.resolve()
+ASSETS_DIR = HERE.parent / "assets"
```
Comment (saforem2) [inline aquarel/utils.py:11]: Add path to `aquarel/assets/` for saving sample plots

File: aquarel/utils.py [hunk 4]
```diff
@@ -41,62 +49,80 @@ def _get_themes():
         )
     )
 
-def _sample_plot(theme: Union[Theme, str], save_as: str = None):
+def make_graph():
```
Comment (saforem2) [inline aquarel/utils.py:52]: Pull out plot creation logic into separate function

File: aquarel/utils.py [hunk 5]
```diff
@@ -41,62 +49,80 @@ def _get_themes():
...
             fig.savefig(
                 save_as,
                 dpi=75,
-                transparent=False,
-                facecolor=fig.get_facecolor(),
                 bbox_inches="tight",
+                facecolor=facecolor,
+                transparent=transparent,
             )
+        else:
```
Comment (saforem2) [inline aquarel/utils.py:111]: Save as `aquarel/assets/{theme.info.name}.png` if not specified

File: aquarel/utils.py [hunk 6]
```diff
@@ -41,62 +49,80 @@ def _get_themes():
...
-        )
-        sns.kdeplot(
-            x="Waiting", y="Duration", hue="Kind", data=geysers, ax=ax[1], fill=True
-        )
-        sns.lineplot(x="Waiting", y="Duration", hue="Kind", data=geysers, ax=ax[2])
-        plt.suptitle("Geysers")
-
+        fig, _ = make_graph()
+        transparent = theme.params.get('savefig.transparent', False)
+        facecolor = theme.params.get('savefig.facecolor', None)
```
Comment (saforem2) [inline aquarel/utils.py:102]: get transparency and facecolor from `theme.params`, if set

**Review by saforem2 — COMMENTED** [review]
👍

**Comment by saforem2** [comment]
@lgienapp would you mind reviewing / pulling this in ?

**Review by lgienapp — APPROVED** [review]

**Review by lgienapp — APPROVED** [review]

**Comment by lgienapp** [comment]
Merged :) thank you for your nice contributions!

**Pull request merged**